<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="5" hostname="DESKTOP-TL1QD2K" failures="4" tests="31" name="correction test" time="3.466" errors="0" timestamp="2020-12-02T16:56:55 CET">
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="additionTest" time="0.018"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="additionTest" time="0.001"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="additionTest" time="0.003"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="additionTest" time="0.0"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="additionTest" time="0.001"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="additionTest" time="0.0"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="divisionTest" time="0.002">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest.divisionTest(CalculetteTest.java:33)
at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:181)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at bstorm.akimt.demoTestNG.correction.prog.ExoProg.main(ExoProg.java:38)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- divisionTest -->
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="divisionTest" time="0.0">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest.divisionTest(CalculetteTest.java:33)
at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:181)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at bstorm.akimt.demoTestNG.correction.prog.ExoProg.main(ExoProg.java:38)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- divisionTest -->
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="divisionTest" time="0.001">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest.divisionTest(CalculetteTest.java:33)
at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:181)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at bstorm.akimt.demoTestNG.correction.prog.ExoProg.main(ExoProg.java:38)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- divisionTest -->
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="divisionTest" time="0.0">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest.divisionTest(CalculetteTest.java:33)
at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:181)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at bstorm.akimt.demoTestNG.correction.prog.ExoProg.main(ExoProg.java:38)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- divisionTest -->
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="divisionTestException" time="0.001"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="multiplicationTest" time="0.001"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="multiplicationTest" time="0.001"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="multiplicationTest" time="0.001"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="multiplicationTest" time="0.001"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="multiplicationTest" time="0.001"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="multiplicationTest" time="0.001"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="multiplicationTest" time="0.001"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="soustractionTest" time="0.002"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="soustractionTest" time="0.0"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="soustractionTest" time="0.001"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="soustractionTest" time="0.002"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="soustractionTest" time="0.002"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="soustractionTest" time="0.001"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.des.LancerDesTest" name="testDCustom_BonusException_intervalle_inf" time="0.007"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.des.LancerDesTest" name="testDCustom_BonusException_intervalle_sup" time="0.0"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.des.LancerDesTest" name="testDCustom_NbrFaceException_pair" time="0.008"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.des.LancerDesTest" name="testDCustom_NbrFaceException_strict_positif" time="0.001"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.des.LancerDesTest" name="testDCustom_NbrLancerException" time="0.0"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="operationTestException" time="0.0"/>
  <testcase classname="bstorm.akimt.demoTestNG.correction.calculette.CalculetteTest" name="operationTest" time="0.0">
    <skipped/>
  </testcase> <!-- operationTest -->
  <testcase classname="bstorm.akimt.demoTestNG.correction.des.LancerDesTest" name="testD4" time="0.0">
    <ignored/>
  </testcase> <!-- testD4 -->
  <testcase classname="bstorm.akimt.demoTestNG.correction.des.LancerDesTest" name="testDCustom" time="0.0">
    <ignored/>
  </testcase> <!-- testDCustom -->
  <testcase classname="bstorm.akimt.demoTestNG.correction.des.LancerDesTest" name="testD20" time="0.0">
    <ignored/>
  </testcase> <!-- testD20 -->
  <testcase classname="bstorm.akimt.demoTestNG.correction.des.LancerDesTest" name="testD100" time="0.0">
    <ignored/>
  </testcase> <!-- testD100 -->
  <testcase classname="bstorm.akimt.demoTestNG.correction.des.LancerDesTest" name="testD6" time="0.0">
    <ignored/>
  </testcase> <!-- testD6 -->
</testsuite> <!-- correction test -->
